packages/autorest.python/autorest/codegen/models/primitive_types.py

```diff
@@ -122,13 +122,13 @@ def serialization_type(self) -> str:
         return "IO"
 
     def docstring_type(self, **kwargs: Any) -> str:
-        return "AsyncIterator[bytes]" if kwargs.pop("async_mode") else "Iterator[bytes]"
+        return "AsyncIterator[bytes]" if kwargs.get("async_mode") else "Iterator[bytes]"
```

Comment:
To avoid similar problems, I think it is better to replace all `pop` with `get` for docstring or annotation